File: homework.py
import random

# Item e: Creating a simple list
my_list = [10, 20, 30, 40, 50]
print("Initial list elements:", my_list)

# Item g: Adding random numbers to the list
# Generates 5 random numbers between 1 and 100
my_list = [random.randint(1, 100) for _ in range(5)]
print("List with random numbers:", my_list)

# Item i: Sorting the list in ascending order
my_list.sort()
print("Sorted list:", my_list)

File: homework.cpp
#include <iostream>
#include <vector>
#include <algorithm>
#include <ctime>

int main() {
    // Item e: Creating a simple list (vector)
    std::vector<int> my_list = {10, 20, 30, 40, 50};
    
    // Item g: Adding random numbers
    std::srand(std::time(0));
    for(int i = 0; i < 5; i++) {
        my_list.push_back(std::rand() % 100 + 1);
    }

    // Item i: Sorting the list
    std::sort(my_list.begin(), my_list.end());

    std::cout << "Sorted C++ List: ";
    for(int x : my_list) {
        std::cout << x << " ";
    }
    std::cout << std::endl;
    
    return 0;
}